scripts/setupTest.ts

```diff
@@ -3,9 +3,9 @@
 import { vi } from 'vitest';
 import type { Navigation, Page } from '@sveltejs/kit';
 import { readable } from 'svelte/store';
-import * as environment from '$app/environment';
-import * as navigation from '$app/navigation';
-import * as stores from '$app/stores';
+import type * as environment from '$app/environment';
```

Comment:
@huntabyte had to do this to get CI checks to be happy - do you know if this makes sense? have you ran into this before?